Jenkinsfile

```diff
@@ -17,17 +23,20 @@ pipeline {
           branch 'PR-*'
         }
         environment {
-          PREVIEW_VERSION = "0.0.0-SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER"
-          PREVIEW_NAMESPACE = "$APP_NAME-$BRANCH_NAME".toLowerCase()
-          HELM_RELEASE = "$PREVIEW_NAMESPACE".toLowerCase()
+         GATEWAY_HOST = "activiti-cloud-gateway.$PREVIEW_NAMESPACE.35.228.195.195.nip.io"
+         SSO_HOST = "activiti-keycloak.$PREVIEW_NAMESPACE.35.228.195.195.nip.io"
         }
         steps {
           container('maven') {
-           dir ('./charts/activiti-cloud-full-example') {
-	          sh 'make build'
+           dir ("./charts/$APP_NAME") {
+	           // sh 'make build'
+              sh 'make install'
```

Comment:
@almerico Let's also disable concurrent builds in the pipeline to avoid running many tests at the same time

Reply:
@igdianov I will do in separate pr